### PR TITLE
Move url to data variable

### PR DESCRIPTION
**Pull request by rubhanazeem**
Remove hardcoded path and use data variable to get URL.

### CODE REVIEW

**Review by hennevogel — APPROVED** [review]

File: src/api/app/assets/javascripts/webui/request.js [hunk 1]
```diff
@@ -134,11 +134,10 @@ $(document).ready(function(){
 
 function loadDiffs(element){
   $('.loading-diff').removeClass('invisible');
-  var id = element.data('action-id');
-  var number = element.data('req-number');
+  var url = element.data('path');
```
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui/request.js:137]: So it's a path or a URL? :smile:

I would choose one name, either `url` or `path` and stick to it.

File: src/api/app/assets/javascripts/webui/request.js [hunk 2]
```diff
@@ -134,11 +134,10 @@ $(document).ready(function(){
...
   var index = element.data('index');
   var diffLimit = $('.sourcediff').data('diff-limit');
-  var url = '/request/'+ number + '/request_action/' + id + '?index=' + index;
+  url = url + '?index=' + index;
```
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui/request.js:140]: We can combine the lines 137 and 140.

```suggestion
var url = element.data('url') + '?index=' + index;
```

File: src/api/app/assets/javascripts/webui/request.js (hunk 1 above)
Comment (rubhanazeem) [reply src/api/app/assets/javascripts/webui/request.js:137]: Haha, it's the complete URL. I'll update it. :laughing:

File: src/api/app/assets/javascripts/webui/request.js (hunk 2 above)
Comment (rubhanazeem) [reply src/api/app/assets/javascripts/webui/request.js]: Done.

File: src/api/app/assets/javascripts/webui/request.js [hunk 3]
```diff
@@ -134,11 +134,9 @@ $(document).ready(function(){
 
 function loadDiffs(element){
   $('.loading-diff').removeClass('invisible');
-  var id = element.data('action-id');
-  var number = element.data('req-number');
+  var url = element.data('url') + '?index=' + index;
```
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui/request.js:137]: `index` isn't defined yet, it's defined on the line below.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**